plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1908,6 +1909,18 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
                         "%s cannot be downgraded from %d to %d", WRITER_VERSION_PROPERTY, currentProtocolEntry.getMinWriterVersion(), writerVersion.get()));
             }
         }
+        long createdTime = Instant.now().toEpochMilli();
+
+        Map<String, String> existingConfiguration = handle.getMetadataEntry().getConfiguration();
+        Map<String, String> configuration = new HashMap<>(existingConfiguration);
```

Comment:
nit: bring `configuration` inside the subsequent `if` - this is where it is only used (at the moment).